Rakefile

```diff
@@ -166,4 +171,17 @@ def reset
 
   puts "Databases reset"
 end
+
+def elastic_search
+  vagrant_loc = ENV["VAGRANT_KEY_LOCATION"]
+  raise ArgumentError, "Environment variable VAGRANT_KEY_LOCATION not set" if vagrant_loc.nil? || vagrant_loc.empty?
+
+  vagrant_key = File.join(vagrant_loc, "private_key")
+  cmd = "ssh -i #{vagrant_key} vagrant@192.168.33.11 'cd /vagrant/waste-carriers-service/bin && export PATH=\"$HOME/.rbenv/bin:$PATH\" && eval \"$(rbenv init -)\" && curl --ipv4 --silent -X POST http://localhost:9091/tasks/location'"
```

Comment:
This isn't the command I use to reindex elasticsearch. I believe it's done via a bash script, and I've used it often in my VM

```bash
cd /vagrant/waste-carriers-service/bin/
WCRS_SERVICES_ADMIN_PORT=9091 . reindex_elasticsearch.sh
```
`http://localhost:9091/tasks/location` is what you call when you want to read in the CSV file of postcode to lat & long mappings.

`http://localhost:9091/tasks/indexer -d '@registration_mapping.json` I believe is what you call when you want to update the schema in elasticsearch, though I think it does also then reindex.

So I'd try just `ssh -i #{vagrant_key} vagrant@192.168.33.11 'cd /vagrant/waste-carriers-service/bin && WCRS_SERVICES_ADMIN_PORT=9091 . reindex_elasticsearch.sh'`